src/MongoDB.AspNetCore.OData/MongoDB.AspNetCore.OData.csproj

```diff
@@ -4,6 +4,13 @@
     <TargetFramework>net5.0</TargetFramework>
     <OutputType>Library</OutputType>
     <NoDefaultLaunchSettingsFile>true</NoDefaultLaunchSettingsFile>
+    <IsPackable>true</IsPackable>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <AssemblyTitle>MongoDB.AspNetCore.OData</AssemblyTitle>
+    <Product>MongoDB.AspNetCore.OData</Product>
+    <Description>Official MongoDB supported integration library for ASP.NET Core OData server side library.</Description>
```

Comment:
Maybe we can think of a more precise description. Something like:
`<Description>Official MongoDB integration library for ASP.NET Core OData.</Description>`